Damage.cs

```diff
@@ -17,49 +16,54 @@ public EDamageTypes DamageType
                 return damageType;
             }
         }
-        #endregion
 
-        #region TargetAttribute
         public struct Target
         {
             public float Armor;
 
             public ERA ERAData;
+
+            public float Health;
         }
 
         public struct ERA
         {
-            public float currentValue;
+            public float CurrentValue;
```

Comment:
This is my opinion but wouldn't it be better to leave its name to `Value` because there is no other variable which means an old or stored value? Just my two-cents. I might be wrong.